news/2017-08-17-happystick-summer-tour-concludes.md

```diff
@@ -0,0 +1,50 @@
+
+[Full match information](https://osu.ppy.sh/community/matches/35244291)
+
+
+<p align="center"><iframe src="https://player.twitch.tv/?video=v167003179&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="620"></iframe></p>
+  
+  
+The [12000 - 18000](http://challonge.com/pxrd8vej) Grand Final told a fascinating story. On one side was [Cyndere](https://osu.ppy.sh/u/5477793), the bracket’s Spring Tour champion reprising his performance by making his way through the Winners Bracket quickly and efficiently. On the other was [egedi](https://osu.ppy.sh/u/3975137), who lost in a tiebreaker in the first round of the brackets before fighting his way through six rounds of the Losers Bracket to make it all the way back to the Grand Final. The two were evenly matched throughout the first best of 11 series, going back and forth showing off their skills. Highlights included egedi’s Full Combo of nomod pick [xi - Akasha](https://osu.ppy.sh/b/931860) and an incredibly tense match on another nomod selection [MISATO - Necro Fantasia](https://osu.ppy.sh/b/1097543) in which egedi would emerge the victor by a mere 620,000 points after both players [broke on the exact same note](https://www.twitch.tv/videos/167003179?t=1h10m35s) (a slider-velocity slowdown following an incredibly intense section of the song) after building up over 1100 combo. The set culminated in a somewhat anticlimactic tiebreaker on [Demetori - Fall of Fall ~ The Door Into Summer](https://osu.ppy.sh/b/1177137) as egedi ran away to a dominating victory to claim the first series 6-5. In the bracket reset match, egedi showed off his incredible aim and consistency to jump out to a fast 5-1 lead to hold multiple championship points. Cyndere brought it back to 5-3 after a nail-biting win on [THE ORAL CIGARETTES - Kyouran Hey Kids!!](https://osu.ppy.sh/b/815857), but egedi was not to be denied. Selecting DT map [Horie Yui - Vanilla Salt](https://osu.ppy.sh/b/320981) (surprisingly, as Cyndere showed off a convincing win on the same map in the first best of 11), egedi put on an inspired performance, missing only one note en route to victory on the map and with it a 6-3 championship win.
```

Comment:
new paragraph at 'The two were evenly matched'

new paragraph at 'The set culminated in a somewhat'

new paragraph at 'Selecting DT map'